modules/owner/owners.rb

```diff
@@ -1,46 +1,59 @@
+# frozen_string_literal: true
+
 # Copyright Erisa A. (erisa.moe) 2016-2020
 module YuukiBot
-    module Owner
-
-        $cbot.add_command(:owners,
-            code: proc { |event, _|
-                event << 'This bot instance is managed/owned by the following users. Please contact them for any issues.'
-                event << "- **#{event.bot.user(YuukiBot.config['master_owner']).distinct}** [**MAIN**]" unless YuukiBot.config['master_owner'].nil?
-                Helper.owners.each {|x|
-                event.bot.user(x).nil? ? event << "- Unknown User (ID: `#{x}`)" : event << "- **#{event.bot.user(x).distinct}**"
-                }
-            },
-            triggers: ['owners']
-        )
+  module Owner
+    YuukiBot.crb.add_command(
+      :owners,
+      code: proc { |event, _|
+        event << 'This bot instance is managed/owned by the following users. ' \
+          'Please contact them for any issues.'
```

Comment:
Maybe this could be reworded? Joining the support server or DMing the bot should be the recommended course of action if issues are encountered.